frontend/src/components/NavBar.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/button-has-type */
+// Pour Romain ou Justine : Si je n'ai pas la ligne pour désactiver le linter, j'ai une erreur eslint qui me dit que le type du bouton doit être défini. Je ne sais pas pourquoi, mais je n'ai pas eu ce problème avec le code de base. Je n'ai pas trouvé de solution pour le moment.
```

Comment:
so put a type to your button (and remove this eslint line)

https://html.com/attributes/button-type/